cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -635,11 +635,9 @@ pub mod bridging {
 		use super::*;
 
 		parameter_types! {
-			/// User fee for ERC20 token transfer back to Ethereum.
-			/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/400 and fee_per_gas 20 GWEI = 2200698000000 + *25%)
-			/// Needs to be more than fee calculated from DefaultFeeConfig FeeConfigRecord in snowbridge:parachain/pallets/outbound-queue/src/lib.rs
-			/// Polkadot uses 10 decimals, Kusama and Rococo 12 decimals.
-			pub const DefaultBridgeHubEthereumBaseFee: Balance = 2_750_872_500_000;
+			/// User fee for delivery cost on bridge hub. calculated by OutboundQueue::calculate_local_fee
+			/// The message will be delivered to bridge hub and cached into outbound queue
+			pub const DefaultBridgeHubEthereumBaseFee: Balance = 4_000_000_000;
```

Comment:
Remove the remote_fee but leave the local_fee here to avoid the OutboundQueue spammed.